docs/src/pages/components/date-range-picker/CustomDateRangePickerDay.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import { makeStyles } from '@material-ui/core';
+import Box from '@material-ui/core/Box';
+import TextField from '@material-ui/core/TextField';
+import AdapterDateFns from '@material-ui/lab/AdapterDateFns';
+import LocalizaitonProvider from '@material-ui/lab/LocalizationProvider';
+import DateRangePicker, { DateRange } from '@material-ui/lab/DateRangePicker';
+import DateRangePickerDay, {
+  DateRangePickerDayProps,
+} from '@material-ui/lab/DateRangePickerDay';
+import clsx from 'clsx';
+
+const useStyles = makeStyles((theme) => ({
```

Comment:
There is a bug with the transition duration on this customization:

![transition bug](https://user-images.githubusercontent.com/3165635/115149412-433a4c80-a064-11eb-828d-484924a19a16.gif)

https://deploy-preview-25812--material-ui.netlify.app/components/date-range-picker/#customized-day-rendering

Reply:
what is the bug?

Reply:
The transition duration is different, leading to an uncomfortable visual feedback

<img width="177" alt="Screenshot 2021-04-20 at 01 00 49" src="https://user-images.githubusercontent.com/3165635/115313618-e75ee900-a173-11eb-8fe7-1c04e25f4cd5.png">

Reply:
Hard to tell whether this is the component or the demo. I'm adding it for now so that we have a reference demo and then we can follow-up when we get to the DateRangePicker.